crates/holochain/src/core/workflow/sys_validation_workflow/validation_deps.rs

```diff
@@ -1,29 +1,53 @@
-use holo_hash::ActionHash;
+use holo_hash::HoloHash;
 use holochain_cascade::CascadeSource;
-use holochain_zome_types::{
-    record::{Record, SignedActionHashed},
-    Action,
+use holochain_types::prelude::*;
+use std::{
+    collections::{HashMap, HashSet},
+    sync::Arc,
 };
-use std::collections::{HashMap, HashSet};
+
+#[derive(Clone)]
+pub struct ValDeps {
+    /// Dependencies found in the same DHT as the dependent
+    pub same_dht: Arc<parking_lot::Mutex<ValidationDependencies<SignedActionHashed>>>,
+    /// Dependencies found in the DPKI service's Deepkey DNA
+    /// (this is not generic for all possible DPKI implementations, only Deepkey)
+    pub deepkey_dht: Arc<parking_lot::Mutex<ValidationDependencies<EntryHashed>>>,
+}
+
+impl Default for ValDeps {
+    fn default() -> Self {
+        Self {
+            same_dht: Arc::new(parking_lot::Mutex::new(ValidationDependencies::new())),
+            deepkey_dht: Arc::new(parking_lot::Mutex::new(ValidationDependencies::new())),
+        }
+    }
+}
```

Comment:
I wound up not using this after all. We might need it if for more robust validation retrying, but things *seem* to be working as they are. I vote to keep it in in case it's needed.

Reply:
Not following because the type is used, is the `deepkey_dht` field not used or something like that?

Reply:
It does look like that's the case, I'm not following why not though?

Reply:
Yes, deepkey_dht is not used. I thought we were going to have to await deepkey items but everything seems to be working fine as is, and I didn't give much thought as to why.